src/ui/components/shared/SharingModal/ReplayLink.tsx

```diff
@@ -26,12 +27,10 @@ export function CopyButton({ recording }: { recording: Recording }) {
 
   return (
     <div className="copy-link relative flex flex-shrink-0 flex-col items-center">
-      <button className={styles.copyURL} {...{ onClick }}>
-        <div className="flex">
-          <Icon filename="doc" size="medium" className={styles.copyIcon} />
-          <div>Copy URL</div>
-        </div>
-      </button>
```

Comment:
The styling for this button was pretty broken before. It was a black background button that turned bright blue background on hover. Now it's our standard primary button style.